docs/upgrading/6-to-7.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: sub-navigation
+order: 1
+title: Upgrading from v6 to v7
+description: v7.0.0 contains a number of breaking changes.
+eleventyNavigation:
+  parent: Upgrading
+---
+
+## Requirements
+
+The minimum requirements for this version have changed:
+
+- [Node.js v22](https://nodejs.org/en/blog/announcements/v22-release-announce)
+- [Eleventy v3](https://github.com/11ty/eleventy/releases/tag/v3.0.0)
+
+## ES modules
+
+The plugin is now provided as an ES module with a named export, instead of using the older CommonJS format (which uses `require()` to import modules).
+
+If you are already using ES modules in your project, update the following import:
+
+```diff
+- import govukEleventyPlugin from '@x-govuk/govuk-eleventy-plugin'
++ import { govukEleventyPlugin } from '@x-govuk/govuk-eleventy-plugin'
+```
+
+If your project is using CommonJS, you will need to import the plugin using a dynamic `import()`. Refer to [CommonJS vs. ESM](https://www.11ty.dev/docs/cjs-esm/) on the Eleventy website for more information.
+
+## Virtual templates
+
+The plugin now provides its layouts using Eleventy’s [virtual templates](https://www.11ty.dev/docs/virtual-templates/) feature.
+
+This means that [default layouts can be overridden](/layouts/#overriding-layouts) in your project by creating layouts with the same name.
+
+Remove the path to the layouts folder in `node_modules` from `dir.layouts` in your configuration file:
+
+```diff
+  dir: {
+    input: 'docs',
+-   layouts: '../node_modules/@x-govuk/govuk-eleventy-plugin/layouts'
+  }
+```
+
+## GOV.UK rebrand
+
+Sites can implement GOV.UK’s refreshed brand from 25 June 2025, using the new `rebrand` option.
+
+The default value for this option is `false`. A subsequent minor release after 25 June 2025 will change this default to `true`.
+
+## Service navigation
+
+The plugin now accepts all options for [service navigation](https://design-system.service.gov.uk/components/service-navigation/), if shown.
+
+The name of this option has changed from `navigation` to `serviceNavigation`, with `navigation.items` renamed `serviceNavigation.navigation`:
+
+```diff
+  import { govukEleventyPlugin } from '@x-govuk/govuk-eleventy-plugin'
+
+  export default function(eleventyConfig) {
+    eleventyConfig.addPlugin(govukEleventyPlugin, {
+-     navigation: {
+-       items: [{
++     serviceNavigation: {
++       navigation: {
+          text: 'API reference',
+          href: '/api'
+        }, {
+          text: 'Glossary',
+          href: '/glossary'
+        }]
+      }
+    })
+
+    // Other configuration options
+    ...
+  }
+```
+
+It is also possible to show a search field in the service navigation, using the new `serviceNavigation.search` option.
+
+## RSS feeds
+
+The RSS feed layout now correctly supports limiting the number of items, in reverse chronological order:
+
+```diff
+  ---
+  eleventyExcludeFromCollections: true
+  layout: feed
+  permalink: /posts/feed.xml
+  pagination:
```

Comment:
Looks like this line needs to change from `pagination` to `collection`:

```suggestion
-  pagination:
+  collection:
```

Reply:
Ooh, good spot. Fixed!